plonky2/src/fri/mod.rs

```diff
@@ -30,6 +30,20 @@ impl FriConfig {
     pub fn rate(&self) -> f64 {
         1.0 / ((1 << self.rate_bits) as f64)
     }
+
+    pub fn fri_params(&self, degree_bits: usize, hiding: bool) -> FriParams {
```

Comment:
There's also a `StarkConfig::fri_params` that could call this

Reply:
Oh I must have missed this one. I think it's kind of natural to have a way to generate the `FriParams` directly from the `FriConfig` though. It also makes the `CircuitBuilder::fri_params` function more succinct.
 I've similarly changed `StarkConfig::fri_params` to call `FriConfig::fri_params`.